### DISCUSSION TITLE
Affected by libwebp vulnerability? Aka how does the build process work?

### DISCUSSION THREAD

**Discussion by curtisy1** [dlemstra/Magick.Native#32] [Q&A] [answered]
Pretty much the same question as [on the `ImageMagick` repo](https://github.com/ImageMagick/ImageMagick/discussions/6650). I was wondering how `libwebp` is bundled currently and whether this library might be affected by the recent `libwebp` vulnerability.

Looking at the build process, I'm guessing it comes from `ImageMagick` itself, so as long as `libwebp` is updated on the system, it should be fine?

**Comment by dlemstra** [comment, marked as answer]
This library uses static dependencies and `libwebp` was updated yesterday in the ImageMagick repository. After the next release of ImageMagick a new release of Magick.NET and magick-wasm will be published. This should happen some time this week.
